docs/user-guide/horizontal-pod-autoscaling/index.md

```diff
@@ -75,9 +75,36 @@ i.e. you cannot bind a horizontal pod autoscaler to a replication controller and
 The reason this doesn't work is that when rolling update creates a new replication controller,
 the horizontal pod autoscaler will not be bound to the new replication controller.
 
+## Changes in the beta1 and v1 specs
+
+As any Kubernetes object you can declare and create a autoscaler in a yaml file. Take into account that the spec changed a little during the stabilization so now instead of writing:
+
+```yaml
+apiVersion: extensions/v1beta1
+[...]
+spec:
+  scaleRef:
+  [...]
+  cpuUtilization:
+    targetPercentage: 50
+```
+
+You write:
+
+```yaml
```

Comment:
Since you have created a yaml file with the contents that the user wants, here, why not just include it directly? 

{% include code.html language="yaml" file="hpa-php-apache-v1.yaml" ghlink="/docs/user-guide/horizontal-pod-autoscaling/hpa-php-apache-v1.yaml" %}